Marlin/ubl_motion.cpp

```diff
@@ -589,6 +589,10 @@
 
       // Otherwise perform per-segment leveling
 
+      #if ENABLED(ENABLE_LEVELING_FADE_HEIGHT)
+        const float fade_scaling_factor = ubl.fade_scaling_factor_for_z(ltarget[Z_AXIS]);
+      #endif
```

Comment:
I removed these lines because the line below (`z_cxcy *= fade_scaling_factor`) was not present, thus `fade_scaling_factor` was unused. Glad it is now restored!